spec/controllers/passwords_controller_spec.rb

```diff
@@ -0,0 +1,138 @@
+require 'spec_helper'
+
+describe PasswordsController do
+  describe "POST request_token" do
+    before { UserMailer.deliveries.clear }
+
+    context "with blank email address" do
+      it "flashes the error message" do
+        post :request_token, email: ""
+        expect(flash.now[:danger]).to eq("Email cannot be blank!")
```

Comment:
You may want to just expect it to `be_present`. Otherwise if you change your message you have to change your spec too. If you don't mind that, then its okay this way as well.